src/components/profilelist/ProfileList.tsx

```diff
@@ -257,12 +282,13 @@ const ProfileList: React.FC = () => {
 
               return (
                 <Profile
+                  unsavedChanges={unsavedProfiles.has(profile.timeslot.pk)}
                   exists
                   active={profile.active}
                   filters={filters || new Map<FilterPK, Filter>()}
                   timeslots={timeslots || new Map<TimeslotPK, Timeslot>()}
                   isTimeslotInUse={(timeslot: Timeslot): boolean => usedTimeslots.has(timeslot.name)}
-                  key={profile.pk}
+                  key={JSON.stringify(profile)}
```

Comment:
This is not the best way to do things, but it works and is simple. Shouldn't be a problem unless lots of profiles are in use.

Reply:
This has been replaced, so it's not relevant anymore.